ManageIQ/manageiq
# Data Transfer Object (DTO) refresh

Data Transfer Object (Dto) refresh will bring a way where we can write refresh parser, that does not depend on the order of parsing of the APIs (therefore, can be parsed in parallel) and a generic saving code that is able to save the tables based on the dependencies (and can also be saved in parallel)

Steps for Testing/QA [Optional]
-------------------------------

The refresh of Amazon Network manager should not change.
